File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/account_risk_check_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class AccountRiskCheckForm:
    """账号风险检测请求"""
    user_id: str = None
    device_id: str = None
    ip_address: str = None
    scene: str = None
    location: str = None

File: sdkwork-app-sdk-csharp/Models/AccountRiskCheckForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class AccountRiskCheckForm
    {
        public string? UserId { get; set; }
        public string? DeviceId { get; set; }
        public string? IpAddress { get; set; }
        public string? Scene { get; set; }
        public string? Location { get; set; }
    }
}
